tests/installation/ipxe_install.pm

```diff
@@ -148,8 +158,27 @@ sub enter_o3_ipxe_boot_entry {
 }
 
 sub set_bootscript_cmdline_extra {
-
     my $cmdline_extra = " ";
+    if (is_sle('>=16')) {
```

Comment:
`ipxe_install` supports also leap and agama is used for leap 16  as well. I would avoid usage if `is_sle` and would create new utils function `is_agama`. To be distro  and version agnostic, but installer specific.

Reply:
For booting other specific backends when excluding logic we have use `AGAMA=1`. We don't use it for anything else, for logic in our test or not even booting in qemu, so we din't wrap that setting in a function, but feel free.

Reply:
I would say presence of variables `AGAMA` or `AGAMA_AUTO` should indicate agama installer.